cms/templates/js/add-list-item.underscore

```diff
@@ -0,0 +1,3 @@
+<button class="action action-add">
```

Comment:
Yes, I'd rather see it inlined if possible. Having lots of very small template files just makes the product more fragmented and harder to understand IMO.